internal/app/init/pkg/system/runner/containerd/import.go

```diff
@@ -37,39 +38,43 @@ func Import(namespace string, reqs ...*ImportRequest) (err error) {
 	// nolint: errcheck
 	defer client.Close()
 
-	var wg sync.WaitGroup
-
-	wg.Add(len(reqs))
+	errCh := make(chan error)
+	var result *multierror.Error
 
 	for _, req := range reqs {
-		go func(wg *sync.WaitGroup, r *ImportRequest) {
-			defer wg.Done()
-
-			tarball, err := os.Open(r.Path)
-			if err != nil {
-				panic(err)
-			}
+		go func(errCh chan<- error, r *ImportRequest) {
+			errCh <- func() error {
 
-			imgs, err := client.Import(ctx, tarball, r.Options...)
-			if err != nil {
-				panic(err)
-			}
-			if err = tarball.Close(); err != nil {
-				panic(err)
-			}
+				tarball, err := os.Open(r.Path)
+				if err != nil {
```

Comment:
there's another issue I missed while doing review. this is actually _changing_ `err` as return argument of the outer function, this is not clean, I'll submit follow-up PR. 

I wonder why linters don't catch that...

Reply:
Yeah the linting has been spotty. Something to look into.